### ISSUE TITLE
Broke my IDE

### ISSUE THREAD

**Issue by fetzu** [cs50/v4-issues#58]
#### What symptom did you see (attaching screenshots as necessary)?
I'm an idiot and inadvertently deleted all files (including ./c9, etc...) while running a git command. I am now greeted with the following screen:
![image](https://user-images.githubusercontent.com/6372605/110832227-586cd000-829b-11eb-9293-f2d2fe0c1db1.png)

Navigating through the broken CSS to the installation then gives me a console (output) I cannot scroll in and an error ("AWS Cloud 9 Installer - Installation failed") like this:
![image](https://user-images.githubusercontent.com/6372605/110832622-cdd8a080-829b-11eb-919e-b191ad28a0a2.png)

#### When date and time (UTC) did the problem happen?
Thursday, 11 March 2021, 17:30 UTC

#### What steps did you follow to reproduce the problem?
1. I just navigate to https://ide.cs50.io
2. I tried with another browser (Edge, the chromium flavor)
3. I tried deauthorizing the IDE from my GitHub account, as well as clearing the cache and cookies in my main browser.

#### Which web browser did you use (including version number)?
Firefox 86.0

#### Which operating system did you use (including version number)?
Windows 10 Pro / 20H2 / 19042.867

#### Did you log in to CS50 IDE using the same GitHub account you used to create this issue?
Yes.

**tl;dr: I'm an idiot and deleted some files/folder I shouldn't have and broke the VM. Could you please restore it to "factory defaults" (fortunately I have a copy of everything I need, so no worries on that front.)**

**Comment by fetzu** [comment]
It seems to have reset and everything's back to normal.

If it doesn't work, hit it a few times.. if that doesn't work, just wait a little ;).
